### PR TITLE
{bio}[GCCcore/8.3.0] cutadapt v1.18, cutadapt v2.7 w/ Python 2.7.16 + 3.7.4

### PR DESCRIPTION
**Pull request by lexming**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by lexming** [comment]
Test report by @lexming
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
login2.cerberus.os - Linux centos linux 7.7.1908, Intel(R) Xeon(R) Gold 6126 CPU @ 2.60GHz, Python 2.7.5
See https://gist.github.com/289039410e4687fd94fa33cdd1aaf4dc for a full test report.

**Comment by migueldiascosta** [comment]
Test report by @migueldiascosta
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
generoso - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2660 v4 @ 2.00GHz, Python 3.6.8
See https://gist.github.com/d82aaeb92192661188047d887e5006ba for a full test report.

File: easybuild/easyconfigs/c/cutadapt/cutadapt-2.7-GCCcore-8.3.0-Python-3.7.4.eb [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+        'checksums': ['dcd8f5ef5da5564f514a990573a48a0c347ee1fdbb9b6374d31592819868f7ba'],
+    }),
+    ('dnaio', '0.4.1', {
+        'checksums': ['371a461bac0e821ff52f6235f0de4533ac73b0e990b470e9625486f2e6df2cd7'],
+    }),
+    (name, version, {
+        'checksums': ['070dec8d94b8bda72906c614b9e71bd61254a67a176dd17e5b57671edd567983'],
+    }),
+]
+
```
Comment (migueldiascosta) [inline easybuild/easyconfigs/c/cutadapt/cutadapt-2.7-GCCcore-8.3.0-Python-3.7.4.eb]: no need for the `fix_python_shebang_for` and `sanity_check_commands`?

Comment (lexming) [reply easybuild/easyconfigs/c/cutadapt/cutadapt-2.7-GCCcore-8.3.0-Python-3.7.4.eb]: yep, there is no need, the default `sanity_check_commands` already check the `cutadapt` module and the shebang is ok. Fixed :+1:

**Comment by lexming** [comment]
Test report by @lexming
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
login2.cerberus.os - Linux centos linux 7.7.1908, Intel(R) Xeon(R) Gold 6126 CPU @ 2.60GHz, Python 2.7.5
See https://gist.github.com/13369d1ddc79a53a57d8006f52b44353 for a full test report.

**Comment by migueldiascosta** [comment]
Going in, thanks @lexming!

**Comment by migueldiascosta** [comment]
Going in, thanks @lexming!

File: easybuild/easyconfigs/c/cutadapt/cutadapt-2.7-GCCcore-8.3.0-Python-3.7.4.eb (hunk 1 above)
Comment (boegel) [reply easybuild/easyconfigs/c/cutadapt/cutadapt-2.7-GCCcore-8.3.0-Python-3.7.4.eb]: @lexming Why was `sanity_check_commands` removed? There are no sanity check commands by default (only paths), and they were put in place for a good reason (see #9219)

Comment (lexming) [reply easybuild/easyconfigs/c/cutadapt/cutadapt-2.7-GCCcore-8.3.0-Python-3.7.4.eb]: The `sanity_check_commands` was importing the python module with `python -c 'import cutadapt.seqio'`, which seemed unnecessary as the `cutadapt` module is already imported by default.

Comment (boegel) [reply easybuild/easyconfigs/c/cutadapt/cutadapt-2.7-GCCcore-8.3.0-Python-3.7.4.eb]: That was added in #9219 to catch the missing `xopen`. `import cutadapt` works fine if `xopen` is missing, `import cutadapt.seqio` does not .

And it's better to leave the `--help` test in too, it's a good way to ensure the shebang is correct.

Comment (lexming) [reply easybuild/easyconfigs/c/cutadapt/cutadapt-2.7-GCCcore-8.3.0-Python-3.7.4.eb]: and the previous `sanity_check_commands` were not testing the main module `cutadapt` as far as I remember.

Comment (boegel) [reply easybuild/easyconfigs/c/cutadapt/cutadapt-2.7-GCCcore-8.3.0-Python-3.7.4.eb]: @lexming No, but that's done by default anyway when using `PythonPackage` (and not via `sanity_check_commands`, that's done separately).

Comment (lexming) [reply easybuild/easyconfigs/c/cutadapt/cutadapt-2.7-GCCcore-8.3.0-Python-3.7.4.eb]: `cutadapt.seqio` does not exist in cutadapt v2. It should not have been removed from cutadapt v1 though. Fix in #9869

**Pull request merged**